Kostyukevich/lab4.cpp

```diff
@@ -0,0 +1,93 @@
+#include "windows.h"
+#include "TlHelp32.h"
+#include <fstream>
+#include <iostream>
+#include <string>
+using namespace std;
+
+int main() {
+
+
+	ifstream deadListFile("deadList.txt");
+	ofstream logFile("log.txt");
+
+	TCHAR processName[200][MAX_PATH];
+	string processNameStr[200];
+	TCHAR buff[MAX_PATH];
+
+	char charProcessName[MAX_PATH];
+	int size = 0;
+	
+	while (deadListFile.getline(charProcessName, MAX_PATH)) {
+		processNameStr[size] = charProcessName;
+		mbstowcs(processName[size], charProcessName, strlen(charProcessName) + 1);
```

Comment:
It will not compile if you set project to use Multi-Byte strings.
Besides, it will not compile becaues `mbstowcs` is unsafe, you need to add something to project's settings or to code to make it work. You should use `mbstowcs_s` instead.